src/index.html

```diff
@@ -7,6 +7,377 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header 
+      class="header"
+    >
+      <a 
+        class="header__img" href="#home">
+        <img 
+          src="./images/logo.png" 
+          alt="logo"
+          class="header__logoImg"
+          >
+      </a>
+      <nav
+        class="header__nav"
+      >
+        <ul class="header__ul">
+          <li class="header__list">
+            <a class="header__a is-active" href="#apple">
+              apple
+            </a>
+          </li>
+          <li class="header__list">
+            <a class="header__a" href="#samsung">
+              samsung
+            </a>
+          </li>
+          <li class="header__list">
+            <a class="header__a" href="#smartphones">
+              смартфоны и телефоны
+            </a>
+          </li>
+          <li class="header__list">
+            <a 
+              class="header__a"
+              href="#laptops"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="header__list">
+            <a class="header__a" href="#gadgets">
+              гаджеты
+            </a>
+          </li>
+          <li class="header__list">
+            <a class="header__a" href="#tablets">
+              планшеты
+            </a>
+          </li>
+          <li class="header__list">
+            <a class="header__a" href="#photo">
+              фото
+            </a>
+          </li>
+          <li class="header__list">
+            <a class="header__a" href="#video">
+              видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="main__container">
+        <div 
+          class="main__card main__card--margin"
+          data-qa="card"
+        >
+          <div class="main__div main__div--center">
+            <img 
+            src="./images/imac.jpeg" 
+            alt="picture" 
+            class="main__img"
+            >
+          </div>
+          <h1 class="main__header main__header--margin_bottom">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h1>
+          <h2 class="main__common-text main__common-text--margin_bottom">
+            Код товара: 195434
+          </h2>
+          <div class="main__reviewes main__reviewes--margin_bottom">
```

Comment:
```suggestion
          <div class="main__reviews reviews">
```
why do you need separate class to margin-bottom? If you need set margins, set them in main__reviews and set own styles for reviews block in reviews class.
`main__reviews` describes positioning of element in the `main` block
`review` describes own styles for element